transact/alchemy_simulateExecutionBundle.yaml

```diff
@@ -456,9 +456,10 @@ components:
           items:
             $ref: '#/components/schemas/Log'
         error:
-          type:
-            - string
-            - 'null'
+          type: string
+        revertReason:
+          $ref: '#/components/schemas/revertMsg'
```

Comment:
lets chance to RevertMessage or RevertMsg

Reply:
@bmoyroud this is returned as `revertReason` in the response so kept it the same. example: 

```json
{
    "jsonrpc": "2.0",
    "id": null,
    "result": {
        "changes": [],
        "gasUsed": null,
        "error": {
            "message": "execution reverted",
            "revertReason": "ERC20: transfer amount exceeds allowance"
        }
    }
}

Reply:
im speaking specifically about the `revertMsg` component not `revertReason` as the key in the response object

so something like

```
revertReason:
  $ref: '#/components/schemas/RevertMessage'
```